src/spec/settings_spec.js

```diff
@@ -16,25 +16,31 @@ describe('Settings Page', () => {
         auth.saveToken(permanentToken);
       }));
   }
-  ///This test will be commented until we add the new Domain Functionality
-  //it('should show add domain input when you click in add domain button', () => {
-    // Arrange
-    //beginAuthenticatedSession();
-    //var settings = new SettingsPage();
+  it('should show add domain input when you click in add domain button', () => {
+    //Arrange
+    beginAuthenticatedSession();
+    browser.addMockModule('descartableModule2', () => angular
+      .module('descartableModule2', ['ngMockE2E'])
+      .run($httpBackend => {
+        $httpBackend.whenGET(/\/accounts\/[\w|-]*\/domains/).respond(200, {
+          "domains": [{name: "relay.com"}, {name: "fromdoppler.com", disabled: true }, {name: "makingsense.com", disabled: true }],
+          "defaultDomain": "relay.com"
+        });
+      }));
+    var settings = new SettingsPage();
 
     //Act
-    //browser.get('/#/settings/domain-manager');
+    browser.get('/#/settings/domain-manager');
 
     //Assert
-    //expect(settings.isDomainInputVisible()).toBe(false);
+    expect(settings.isDomainInputVisible()).toBe(false);

```

Comment:
Since it is not the Assertion (in uppercase of this tests) I would remove the comment `// Assert`. I mean, since it is only to validate our setup, in my opinion it is part of the arrange.

Also `browser.get('/#/settings/domain-manager');` is part of Arrange section.